### DISCUSSION TITLE
配置项目的主题能不能加个自动？

### DISCUSSION THREAD

**Discussion by elarbor** [JackySoft/marsview#64] [Ideas]
![image](https://github.com/user-attachments/assets/20eacda2-6d23-40be-963b-1ef1ad7f2f9d)

**Comment by JackySoft** [comment]
自动是什么意思

**Reply by elarbor** [reply]
就是自动识别电脑主题，是暗的还是亮的，然后让平台也显示系统一样的主题
https://sanonz.github.io/2020/to-build-dark-and-light-theme-with-web/
https://www.jianshu.com/p/575557eb1465
